maps/metaclub.dmm

```diff
@@ -42938,7 +42938,7 @@
 	},
 /obj/structure/transit_tube/station{
 	dir = 4;
-	req_access_txt = "32;19;501";
+	req_one_access_txt = "32;19;501";
```

Comment:
I feel the "_txt" part should not be there.

Reply:
you're wrong

Reply:
I don't know shit about accesscode, but I've never seen "req_one_access_txt" before. Only "req_access_txt" or "req_one_access".

Reply:
That would be because you don't know shit about accesscode

Reply:
Precisely what I said.